hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/mapreduce/RegionSizeCalculator.java

```diff
@@ -115,7 +116,7 @@ public long getRegionSize(byte[] regionId) {
     Long size = sizeMap.get(regionId);
     if (size == null) {
       LOG.debug("Unknown region:" + Arrays.toString(regionId));
-      return 0;
+      return UNKNOWN_SIZE;
```

Comment:
Please let's not conflate the two issues here. @frostruan if you want to change the 0-value response behavior, please do that in a dedicated ticket. We should have NPE protection independent of whatever solution is decided upon via HBASE-28399. In the mean time, this ticket can fix the NPE and return 0 in that case, which is consistent with the existing code.

Reply:
Ok, I agree.